tests/ops/op_math/test_decompositions.py

```diff
@@ -193,8 +190,8 @@ def test_zyz_decomposition_jax(self, U, expected_params):
     ),
     # Try a few specific special unitaries
     (I, [0, 0, 0, 0]),  # This triggers the if-conditional trivially
-    (X, [np.pi * 3 / 2, 0.0, 7 * np.pi / 2, -np.pi / 2]),
-    (Y, [np.pi / 2, np.pi, np.pi / 2, -np.pi / 2]),
+    (X, [np.pi / 2, 0.0, np.pi / 2, -np.pi / 2]),
+    (Y, [0.0, np.pi, 0.0, -np.pi / 2]),
```

Comment:
@JerryChen97 The new results are mathematically equivalent but arguably much cleaner.